src/Compilers/Core/Portable/DiagnosticAnalyzer/AnalyzerAssemblyLoader.cs

```diff
@@ -54,31 +52,18 @@ public Assembly LoadFromPath(string fullPath)
 
         #endregion
 
-        private Assembly LoadFromPathUnchecked(string fullPath)
-        {
-            return LoadFromPathUncheckedCore(fullPath);
-        }
-
-        private Assembly LoadFromPathUncheckedCore(string fullPath, AssemblyIdentity identity = null)
+        protected Assembly LoadFromPathUnchecked(string fullPath)
```

Comment:
Please add a doc comment that helps differentiate why you should call this method over `LoadFromPathImpl`. 

Given the relationship between these two methods perhaps consider naming them such that they have an obvious relationship. Example: 

- `LoadFromPathUncheckedImpl(string fullPath)`
- `LoadFromPathUnchecked(string fullPath)`

Reply:
Added some docs and also took your rename since that made things clearer now, especially now that the "Core" method is out of the way.